### PR TITLE
ECPAPPSERVERWEBLOGIC-260 [WIP]

### PR DESCRIPTION
**Pull request by horodchukanton**
DeployApp spec test [wip] (pushed for review)

### CODE REVIEW

File: specs/DeployApp.groovy [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+                 targets : 'AdminServer',
+            ]
+        )
+        """, getResourceName())
+        then:
+        logger.debug(result.logs)
+        assert result.outcome == 'success'
+    }
+
+    def 'Check deployment'() {
```
Comment (imago-storm) [inline specs/DeployApp.groovy]: Please move this into a parent helper, we will need this method often. Create something akin to runWLST: checkPage(url) -> code, text

Comment (imago-storm) [reply specs/DeployApp.groovy]: Also: you cannot rely on the order of features, and you cannot make them dependent. If you want to test deployed application, do this in the "deploy application" feature.

File: specs/DeployApp.groovy [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                 wlstabspath: '$wlstPath',
+                 appname : '$applicationName',
+                 apppath : "$applicationWarPath",
+                 targets : 'AdminServer',
+            ]
+        )
+        """, getResourceName())
+        then:
+        logger.debug(result.logs)
+        assert result.outcome == 'success'
```
Comment (imago-storm) [inline specs/DeployApp.groovy]: Add some logs checks here (regular expressions), and also move URL check here.

**Comment by horodchukanton** [comment]
Closed due to changing base branch (avoiding duplicate merge conflict resolving)
Opened new pull request

**Pull request closed without merging**